### PR TITLE
Education urte

### PR DESCRIPTION
**Pull request by UrteZ**
- added education section
-created education data, js files
- added custom style
- initial responsive features added (but need to fix for final version)

### CODE REVIEW

File: css/custom.css [hunk 1]
```diff
@@ -713,6 +713,73 @@ canvas {
...
+        font-size: 32px; 
+        font-weight: bold;
+    }
+
+    .row.education > .edu-top-section > p { 
+        color: #6c757d;
+        font-size: 18px;
+        margin: 30px 15px 30px 15px;
+    }
+    .row.education > .edu-main-section {
```
Comment (ugniusnor) [inline css/custom.css]: Nesu tikras ar šitas left reikalingas

Comment (UrteZ) [reply css/custom.css]: ok, nuimta

File: css/custom.css [hunk 2]
```diff
@@ -715,25 +715,19 @@ canvas {
 /* education */
 .row.education {
     text-align: center;
-}
-.row.education > .edu-top-section {
```
Comment (Karina-N) [inline css/custom.css]: cia irgi uztenka vienos klases

Comment (UrteZ) [reply css/custom.css]: pakeiciau :)

**Review by Karina-N — APPROVED** [review]

**Review by ugniusnor — APPROVED** [review]
very good very nice, let's make issues for resposive designs

**Review by giedrunas — APPROVED** [review]

**Pull request merged**